### DISCUSSION TITLE
Weird Entry Pages like /ijks/

### DISCUSSION THREAD

**Discussion by danieljbradley** [plausible/analytics#1717] [Anything Else]
About a week ago I started to get weird URL entry pages like:
/ijks/
/yyds/
/iods/

I do not have these URL's on my site.  They bring up a 404 error when I click them.

I also started getting patiofurniturea. shop as a "top source".  I have a clothing business so I have no idea why would link to me.  I think it might be a scam and I am hesitant to go to their site.

What's weirder is that, when I filter on /ijks/ as an Entry Page, it shows that 7 users visited through Facebook and 1 visited through patiofurniturea. shop

Has anybody else seen anything like this?  Thanks in advance for your help
![plausible](https://user-images.githubusercontent.com/44952423/155747857-ac695c0a-88fa-4292-a6e0-b2e1568a4ac9.PNG)
![plausible2](https://user-images.githubusercontent.com/44952423/155747859-6a6f52ea-a590-49f6-8d70-2e82044fda92.PNG)
.

**Comment by metmarkosaric** [comment]
hi @danieljbradley! what website is this for? feel free to [contact our customer support](https://plausible.io/contact) privately so we can explore more. thanks!

**Comment by danieljbradley** [comment]
Marko, I messaged customer support.  I'm thinking it's not a plausible issue.  You guys are just showing the weird traffic we are getting.  If you have some time though I appreciate taking a quick look.

**Comment by SaneChoice** [comment]
Hi @danieljbradley - that patio url suspiciously looks like a bounce redirect. Usually used by people who buy traffic (which I am sure you are not) and they bounce of these fake domains.  Ill keep an eye on my instance as well for this.
